libc-test/build.rs

```diff
@@ -3714,13 +3715,20 @@ fn test_linux(target: &str) {
             // Added in Linux 5.13
             "PTRACE_GET_RSEQ_CONFIGURATION" if sparc64 => true,
 
+            | "MADV_COLD"
+            | "MADV_PAGEOUT"
+            | "MADV_POPULATE_READ"
+            | "MADV_POPULATE_WRITE"
+            if musl || android => true,
```

Comment:
Android isn't tested in test_linux but in test_android: https://github.com/tatref/libc/blob/288de155c495592f8ea4427750444f4cacc0899b/libc-test/build.rs#L1514